<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Database</class>
 <widget class="QDialog" name="Database">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ButtonCancel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>550</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="m_pSections">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>780</width>
     <height>540</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="TextureDatabaseWindow" name="widget">
    <attribute name="title">
     <string>Textures</string>
    </attribute>
   </widget>
   <widget class="ObjectDatabaseWindow" name="Objects">
    <attribute name="title">
     <string>Objects</string>
    </attribute>
   </widget>
   <widget class="GlobalVariableDatabaseWindow" name="m_pGlobalVariables">
    <attribute name="title">
     <string>Global Variables</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="ButtonOK">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>550</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonApply">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>550</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextureDatabaseWindow</class>
   <extends>QWidget</extends>
   <header>texturedatabasewindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ObjectDatabaseWindow</class>
   <extends>QWidget</extends>
   <header>objectdatabasewindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GlobalVariableDatabaseWindow</class>
   <extends>QWidget</extends>
   <header>globalvariabledatabasewindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
